docs/blog/newsletter-september-2017.rst

```diff
@@ -0,0 +1,83 @@
+Stack Overflow Docs Beta Sunsetting and Crowdsourcing Docs
+***********************************************************
+This month, the `Stack Overflow Documentation beta shutdown <https://meta.stackoverflow.com/questions/354217/sunsetting-documentation>`_ prompted a conversation about crowdsourced docs and whether they can ever work well. Folks mentioned `Django <https://docs.djangoproject.com/en/1.11/>`_, `Wikipedia <https://www.wikipedia.org>`_, and `Ubuntu <https://help.ubuntu.com>`_ as examples of effective crowdsourced documentation. And on the otherside you have open-source software companies like OpenOffice and Mozilla, who tried volunteer-only docs but eventually hired writers.
+
+So what *does* it take to make crowdsourced docs work? They seem to work best when companies clearly value documentation – even to the point of requiring developers to write it before they can commit code. Others mentioned that it's vital to provide lots of detail about the content contributions you're looking for. Finding the right model is important too: some companies have all-volunteer-created docs, and others solicit volunteer contributions that are eventually reviewed and curated by an in-house writing team. Crowdsourcing is also a good way to dig up questions that real users have so you can explicitly address them in your documentation.
+
+People also had some interesting perspectives on why the Stack Overflow (SO) effort failed:
+
+* No clear vision communicated for SO documentation
+* Return on investment for SO docs was too low to pursue
+* Docs are a hard sell--it's not easy to get people to volunteer to write or curate them
+* SO shut the Docs beta down before it had time to catch on
+* SO wasn't seen as the "right" place to go for docs (even if a company's official docs aren't well done)
+* No easy or effective way to address gaps in official docs in the SO environment (which is problem:solution-based)
+
+If you're interested in a lengthier dicsussion about the SO Documentation beta sunsetting and crowdsourced docs in general, check out last month's episode of the `Write The Docs podcast <http://podcast.writethedocs.org/2017/08/22/stack-overflow-failure-open-source-challenges/>`_!
+
+*************************************
+Doc Portfolios: A Perpetual Conundrum
+*************************************
+If you're job hunting right now, the likelihood is high that you've spent some time sprucing up your writing portfolio. But much more than, say, a journalist or a graphic designer, documentarians have to tread carefully when assembling our work samples. Much of the work we do is either so context-dependent that sharing our samples might not tell a prospective employer anything useful... or it's straight-up proprietary, in which case we can't share them at all. Maintaining a portfolio that's applicable, fresh, and accessible is a problem that, as one documentarian pointed out this month, is a "perpetual conundrum" we all face.
```

Comment:
maybe "you've probably spent some time..." instead of "the likelihood is high that you've spent some time..."

Reply:
also maybe remove "right now"? How is now different from other times? Don't we always need work samples?

Reply:
Excellent catches, both! My default to 'maximum verbosity' was clearly getting in the way here. ;)